### ISSUE TITLE
 module 'torch.nn.init' has no attribute 'kaiming_normal_'

### ISSUE THREAD

**Issue by subhobrata** [AI6-Bangalore-Chapter/2018-cycle-2#8]
Hi ,

I am trying to run lession8_demo code in colab.

But I am getting below error.

 module 'torch.nn.init' has no attribute 'kaiming_normal_'

learn = ConvLearner.pretrained(f_model, md, metrics=[accuracy])
learn.opt_fn = optim.Adam

Link to my notebook
https://github.com/subhobrata/fastai/blob/master/Copy_of_Lesson8_demo.ipynb

Please help me on this.

Thanks and Regards,
Subho
